src/platfdep.h

```diff
@@ -34,7 +34,6 @@ namespace mmex
     const wxFileName GetSharedDir();
     const wxFileName GetUserDir(bool create);
     const wxFileName GetLogDir(bool create);
-    static const wxFileName GetInstallDir();
```

Comment:
This line (which has been removed) is required for a successful build.

Reply:
We have implemented this function in Windows only and it's must be static and not public.

Reply:
I see that fix provided in calling function which corrects the build. Now OK